octokit/client.py

```diff
@@ -9,11 +9,13 @@
 
 # https://code.google.com/p/uri-templates/wiki/Implementations
 
-from .resources import Resource
+from resources import Resource
```

Comment:
I can't run the init.py file because of that error either. It seems it's because of Python 3, but you and Ho-wei are both using Python 3 right? Maybe it's because you guys are using the virtual env?